control-plane-operator/controllers/hostedcontrolplane/hostedcontrolplane_controller.go

```diff
@@ -4523,6 +4523,10 @@ func (r *HostedControlPlaneReconciler) reconcileCoreIgnitionConfig(ctx context.C
 }
 
 func (r *HostedControlPlaneReconciler) reconcileRouter(ctx context.Context, hcp *hyperv1.HostedControlPlane, releaseImageProvider imageprovider.ReleaseImageProvider, createOrUpdate upsert.CreateOrUpdateFN) error {
+	if hcp.Spec.Platform.Type == hyperv1.IBMCloudPlatform {
```

Comment:
Why isn't this change in the main branch for the `admitHCPManagedRoutes` function?